packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -193,29 +197,29 @@ class DateTimeUtils {
    * @param {string} ambiguousDateTime - The ISO date time with the ambiguous hour.
    * @return {string} - The standard time zone offset display.
    */
-  static getStandardTZDisplay(ambiguousDateTime) {
-    const standardDateTime = moment(ambiguousDateTime);
+  static getStandardTZDisplay(ambiguousDateTime, timeZone) {
+    const standardDateTime = DateTimeUtils.createSafeDate(ambiguousDateTime, timeZone);
     if (!standardDateTime.isValid()) {
       return '';
     }
 
     standardDateTime.add(1, 'days');
-    return standardDateTime.tz(moment.tz.guess()).format('z');
+    return standardDateTime.tz(standardDateTime.tz()).format('z');
   }
 
   /**
    * Gets the long standard time zone offset display. (e.g. America/Chicago CST -6:00)
    * @param {string} ambiguousDateTime - The ISO date time with the ambiguous hour.
    * @return {string} - The long standard time zone offset display.
    */
-  static getStandardExpandedTZDisplay(ambiguousDateTime) {
-    const standardDateTime = moment(ambiguousDateTime);
+  static getStandardExpandedTZDisplay(ambiguousDateTime, timeZone) {
```

Comment:
these in-line docs all need to be updated to include the timeZone parameter

Reply:
updated here b5f07bc5f04b127f3d75bc46a3651a2e1091b165